packages/tinymce/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@web-stories-wp/tinymce",
+  "private": true,
+  "engines": {
+    "node": ">= 15",
+    "npm": "~7"
+  },
+  "exports": {
+    "default": "./src/web-stories.js"
+  },
+  "main": "./src/web-stories.js",
+  "type": "module",
+  "dependencies": {
+    "prop-types": "^15.7.2",
+    "@wordpress/compose": "^3.23.0",
+    "@wordpress/components": "^12.0.7",
+    "@wordpress/data": "^4.26.0",
+    "@wordpress/element": "^2.19.0",
+    "@wordpress/i18n": "^3.17.0"
+  },
+  "devDependencies": {
+    "react": "^16.14.0",
```

Comment:
Why was this added? And not using `@wordpress/element`?

Reply:
React is just used in the tests. So i dont think it is an issue.

Reply:
It becomes an issue if the version of React in `react` is different from the version provided by `@wordpress/element`.

All code - tests and source - should use the same version of the same package, in this case  only `@wordpress/element`

Reply:
Seems like it wasn't need anymore. Removed in https://github.com/google/web-stories-wp/pull/6578/commits/d333406a7a91a0990c55ce938f6021bf8373d889